dev/devicelab/pubspec.yaml

```diff
@@ -8,6 +8,10 @@ environment:
   sdk: ">=2.3.0 <3.0.0"
 
 dependencies:
+  flutter:
+    sdk: flutter
```

Comment:
Hmm, I think this may be an artifact from when I was trying to use flutter-web for this. I'll delete it and see if it still runs OK.

Reply:
Indeed it is - removed...